vendor/k8s.io/kubernetes/pkg/controller/namespace/namespace_controller.go

```diff
@@ -85,6 +85,10 @@ func NewNamespaceController(
 	}
 	ignoredGroupVersionResources := []unversioned.GroupVersionResource{
 		{Group: "", Version: "v1", Resource: "bindings"},
+
+		// FIXME: This should be removed after 1.6 rebase lands?
```

Comment:
If this is meant to be dropped with 1.6, name the commit accordingly: UPSTREAM <drop>...